docs/source/exposing.md

```diff
@@ -0,0 +1,267 @@
+
+#### `PodIP` Type
+
+Address broadcasted to clients/nodes is taken from Pod.
+Because Pod address may be ambiguous, you may provide what's the source by setting `podIP.source`.
+By default, address is taken from Pod `status.PodIP`.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    broadcastOptions:
+       clients:
+         type: PodIP
+         podIP:
+           source: Status
+```
+
+#### `ServiceClusterIP` Type
+
+Address broadcasted to clients/nodes is taken from node dedicated Service, from `spec.ClusterIP` field.
+
+In order to configure it, your `nodeService` template must deploy Service having ClusterIP.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    broadcastOptions:
+       clients:
+         type: ServiceClusterIP
+```
+
+#### `ServiceLoadBalancerIngress` Type
+
+Address broadcasted to clients/nodes is taken from the node dedicated Service, from `status.ingress[0].ipAddress` or `status.ingress[0].hostname` field.
+
+In order to configure it, your `nodeService` template must deploy LoadBalancer Service.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    broadcastOptions:
+       clients:
+         type: ServiceLoadBalancerIngress
+         podIP:
+           source: Status
+```
+
+## Deployment Examples
+
+In this section, we will showcase specific examples of various network scenarios and explain how nodes and clients communicate with one another.
+### In-cluster only
+
+ScyllaCluster definition:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: ClusterIP
+    broadcastOptions:
+      clients:
+        type: ServiceClusterIP
+      nodes:
+        type: ServiceClusterIP
+```
+
+Both client and nodes are deployed within the same Kubernetes cluster. 
+They talk through IP addresses from ClusterIP Service.
+Because ClusterIP Services are only routable within the same Kubernetes cluster, connection from outside is impossible.
+
+![ClusterIPs](static/exposing/clusterip.svg)
+
+### In-cluster node-to-node, VPC clients-to-nodes
+
+ScyllaCluster definition:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: ClusterIP
+    broadcastOptions:
+      clients:
+        type: PodIP
+      nodes:
+        type: ServiceClusterIP
+```
+
+In this scenario, we assume that the Pod IP subnet is routable within a VPC. 
+Clients within the VPC network can communicate directly with ScyllaCluster nodes using Pod IPs.
+Nodes communicate with each other exclusively within the same Kubernetes cluster.
+
+![PodIPs](static/exposing/podips.svg)
+
+### Multi DC
+
+ScyllaCluster definition:
```

Comment:
should we show both DC's and the externalSeeds option? or at least put a link to where multiDC is shown

Reply:
maybe the whole multidc is better to be left to the port of the docs about multidc

Reply:
This document is about exposing options, connecting two clusters - no matter how - is not tied to it. We can add link to page describing it once we have it.

Reply:
I admit I am confused by the answer.

What's the point of showing a setup of 2 ScyllaClusters in 2 Kubernetes cluster without connecting them?

this picture `![MultiDC](static/exposing/multidc.svg)` says multidc :man_shrugging:

Reply:
Remove "region" term in favor of multiple VPCs.

> What's the point of showing a setup of 2 ScyllaClusters in 2 Kubernetes cluster without connecting them?

I don't show how to configure clients either and it doesn't make anyone confused :man_shrugging: 

Again, describing how to connect two clusters is the not point of this document.

Reply:
Is this creating 1 ScyllaCluster (in one of the Kubernetes clusters) or 2 (1 in each)? I don't see that explicitly stated, but the picture suggests 2 distinct ScyllaClusters.
Should the picture mark ScyllaCluster object boundaries?

> Nodes, deployed in these distinct VPCs,

This seems like 2 ScyllaClusters which may lead users to think that this sets up the nodes to communicate with each other, but it doesn't. Can you at least make an explicit disclamer that the nodes won't actually talk to each other without using externalSeeds when bootstraping them?

Reply:
> Should the picture mark ScyllaCluster object boundaries?

There's a kube cluster boundary, seems obvious these are two obvious scyllaclusters.

> This seems like 2 ScyllaClusters which may lead users to think that this sets up the nodes to communicate with each other, but it doesn't. Can you at least make an explicit disclamer that the nodes won't actually talk to each other without using externalSeeds when bootstraping them?

It's mentioned that exposeOptions only enables establishing connection. It doesn't say it establishes it. Although added explicit mention that there's a separate page about connecting two.